### DISCUSSION TITLE
Osu!lazer not working after update

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#14874] [Q&A] [answered]
So i downloaded the new update for Osu!Lazer like usual (on my android), but after downloading the update, i tried opening the game but it will just show me a Black Screen, i tried rebooting my android but it didnt fix i.
Can anyone help

**Comment by sortedcord** [comment, marked as answer]
Mentioned here https://github.com/ppy/osu/discussions/14869#discussioncomment-1393337
